### PR TITLE
Mysql

### PR DESCRIPTION
**Pull request by jdsisco**

### CODE REVIEW

File: QT/Naybrr/Naybrr.pro [hunk 1]
```diff
@@ -15,6 +16,7 @@ DEFINES += QT_DEPRECATED_WARNINGS
 #DEFINES += QT_DISABLE_DEPRECATED_BEFORE=0x060000    # disables all the APIs deprecated before Qt 6.0.0
 
 SOURCES += \
+        db_controller.cpp \
```
Comment (serviux) [inline QT/Naybrr/Naybrr.pro]: remove db controller

File: QT/Naybrr/db_controller.cpp [hunk 2]
```diff
@@ -0,0 +1,139 @@
+#include "db_controller.h"
+
```
Comment (serviux) [inline QT/Naybrr/db_controller.cpp:2]: remove this

File: QT/Naybrr/db_controller.h [hunk 3]
```diff
@@ -0,0 +1,42 @@
+#ifndef DB_CONTROLLER_H
+#define DB_CONTROLLER_H
```
Comment (serviux) [inline QT/Naybrr/db_controller.h:2]: remove this

File: QT/Naybrr/user.cpp [hunk 4]
```diff
@@ -64,6 +65,22 @@ int User::itemCount(QQmlListProperty<Item>* list){
 
 
 //TODO: actually connect to the database
+bool createConnection()
```
Comment (serviux) [inline QT/Naybrr/user.cpp:68]: Remove

File: QT/Naybrr/user.cpp [hunk 5]
```diff
@@ -1,4 +1,5 @@
 #include "user.h"
```
Comment (serviux) [inline QT/Naybrr/user.cpp:1]: remove this

File: .gitignore [hunk 6]
```diff
@@ -1,2 +1,20 @@
-#ignore the build folder
-*/*build*/
+# Distribution / packaging
+.Python
```
Comment (serviux) [inline .gitignore:2]: add */*build*/ back to gitignore

File: QT/Naybrr/user.cpp (hunk 5 above)
Comment (jdsisco) [reply QT/Naybrr/user.cpp:1]: The rest of the file yelled at me because it couldn't recognize User without the header.

File: .gitignore (hunk 6 above)
Comment (jdsisco) [reply .gitignore:2]: build is already in the file (line 4)

**Comment by jdsisco** [comment]
Files have been changed, with comments.

File: QT/Naybrr/user.cpp (hunk 5 above)
Comment (serviux) [reply QT/Naybrr/user.cpp:1]: sorry the line numbers seem off, this seems fine, I might have been referring to any of the changes made for connecting to the db

**Comment by serviux** [comment]
Don't forget to push your changes so I can see the update @jdsisco

**Comment by jdsisco** [comment]
Merge completed, but still says I need approval.

**Review by serviux — APPROVED** [review]

**Pull request merged**